src/main/java/io/jenkins/pluginhealth/scoring/probes/LastCommitDateProbe.java

```diff
@@ -0,0 +1,78 @@
+package io.jenkins.pluginhealth.scoring.probes;
+
+import java.io.File;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+import io.jenkins.pluginhealth.scoring.model.Plugin;
+import io.jenkins.pluginhealth.scoring.model.ProbeResult;
+import io.jenkins.pluginhealth.scoring.model.ResultStatus;
+
+import org.slf4j.Logger;
+import org.eclipse.jgit.revwalk.RevCommit;
+import org.slf4j.LoggerFactory;
+import org.springframework.core.annotation.Order;
+import org.springframework.stereotype.Component;
+import org.eclipse.jgit.api.Git;
+import org.eclipse.jgit.api.errors.GitAPIException;
+
+/**
+ * Using the analysis done by {@link SCMLinkValidationProbe#}, this probe determines the last commit date of the plugin's repository.
+ */
+@Component
+@Order(value = LastCommitDateProbe.ORDER)
+public class LastCommitDateProbe extends Probe {
+    private static final Logger LOGGER = LoggerFactory.getLogger(LastCommitDateProbe.class);
+
+    public static final int ORDER = SCMLinkValidationProbe.ORDER + 1;
+
+    @Override
+    public ProbeResult doApply(Plugin plugin) {
+        final String scmLinkValidationProbeKey = new SCMLinkValidationProbe(null, null).key();
```

Comment:
You shouldn't need to instantiate the probe only to get its `key`. If you need it, please refactor that probe to have access to the key.